plugins/linux-pipewire/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ if(NOT ENABLE_PIPEWIRE)
   return()
 endif()
 
-find_package(PipeWire 0.3.33 REQUIRED)
```

Comment:
Mismatch with the commit title (which has 0.3.50).

To avoid issue with Ubuntu 22.04, maybe `#ifdef` the feature until 22.04 is dropped ?

Reply:
This can't be marked as resolved since we still support Ubuntu 22.04.